code/drasil-example/Drasil/PDController/Body.hs

```diff
@@ -71,6 +71,7 @@ import Language.Drasil.Code
 import Language.Drasil.Printers (PrintingInformation(..), defaultConfiguration)
 import Theory.Drasil (DataDefinition, GenDefn, InstanceModel, TheoryModel)
 import Utils.Drasil
+import Drasil.PDcontroller.Unitals
```

Comment:
This should be `PDController`, not `PDcontroller` (capital C, not lower-case c). This occurs multiple times - that's the reason it cannot be found.

Reply:
oh okay